gammapy/datasets/tests/test_map.py

```diff
@@ -1361,3 +1361,67 @@ def test_downsample_onoff():
     assert downsampled.counts.data.sum() == dataset_onoff.counts.data.sum()
     assert downsampled.counts_off.data.sum() == dataset_onoff.counts_off.data.sum()
     assert_allclose(downsampled.alpha.data, 0.5)
+
+
+def test_compute_flux_spatial():
```

Comment:
I realised this test case is maybe a bit to high level. What about if we start on the `MapEvaluator` level?
So define:
- Constant exposure (filled with ones...) with 2 energy bins
- Point source models with a `ConstSpectralModel`
- Gaussian PSF kernel, created with `PSFKernel.from_gauss(geom=region_geom.to_wc_geom())`
- Create a `MapEvaluator()`
- `MapEvaluator.compute_flux_spatial()` should then return exactly the PSF containment per energy as an array like [0.39, 0.39]...